Packages/MIES/MIES_SweepFormula.ipf

```diff
@@ -253,42 +253,71 @@ Function/WAVE FormulaExecutor(jsonID, [jsonPath])
 	Variable jsonID
 	String jsonPath
 
-	Variable i, numIndices, JSONtype
+	Variable i, numIndices, lastCol, JSONtype
 
 	if(ParamIsDefault(jsonPath))
 		jsonPath = ""
 	endif
 
+	// object evaluation
 	JSONtype = JSON_GetType(jsonID, jsonPath)
 	if(JSONtype == JSON_NUMERIC)
 		Make/FREE out = { JSON_GetVariable(jsonID, jsonPath) }
 		return out
+	elseif(JSONtype == JSON_STRING)
+		Make/FREE/T outT = { JSON_GetString(jsonID, jsonPath) }
+		return outT
+	elseif(JSONtype == JSON_ARRAY)
+		if(JSON_GetArraySize(jsonID, jsonPath) == 0)
+			return $""
+		endif
+		if(JSON_GetType(jsonID, jsonPath + "/0") == JSON_STRING)
+			return JSON_GetTextWave(jsonID, jsonPath)
+		endif
+		WAVE out = JSON_GetWave(jsonID, jsonPath)
+		EXTRACT/FREE/INDX out, dummy, JSON_GetType(jsonID, jsonPath + "/" + num2str(p)) == JSON_OBJECT
+		EXTRACT/FREE dummy, indices, dummy[p] < DimSize(out, ROWS) // extract only first dimension
+		WaveClear dummy
+		numIndices = DimSize(indices, ROWS)
+		if(numIndices == 1 && DimSize(out, ROWS) == 1)
+			return FormulaExecutor(jsonID, jsonPath = jsonPath + "/0", graph = graph) // function
+		elseif(numIndices > 0)
```

Comment:
@t-bo data alignment not defined. Add new data to last dimension. Thus, the top array (`out`) will be the last (!) dimension if an operation needs to be carried out. The array needs to get dynamically expaneded to +1 dimension.